### DISCUSSION TITLE
Any jobs?

### DISCUSSION THREAD

**Discussion by balupton** [Tribler/tribler#5920] [Q&A]
I'm curious to further Tribler and its goals, as well as learn and research what is needed. Although, like many others, my finances have gone to survival with zero savings to benefit from the crypto booms.

Is there any opportunity for employment or subcontracts or bounties here for skilled developers who want to help? Or even community managers, dev ops, etc. So that those assisting in a large capacity can also pay their bills.

# Potential work

Listing of mentions of potential work across the project:

> By the second half of 2021 we should plan to hire outsourcers to completely redesign the GUI to modern standards. The finished GUI will become a major compelling point to release Tribler 8.0. This should be a very good moment to "whitewash" our reputation and start a media campaign.
> https://github.com/Tribler/tribler/issues/5587#issue-709786922

# Requests for work

## @balupton

[LinkedIn Resume](https://balupton.com/linkedin). Eager to:

- do those GUI tasks
- run a youtube channel discussing the research papers and making them easily accessible to the community as well as regular interviews and updates with the team
- organise the [various github issues](https://github.com/Tribler/tribler/discussions/5872#discussioncomment-237197)
- make a bot that tracks the research paper links (makes sure they aren't 404) and updates them with [their research gate papers](https://github.com/Tribler/tribler/discussions/5919)
- whatever else that can help you all work together and get things done, including data entry and secretary work

Goal is to build end-user apps on top of Tribler, one of the MVPs being an app that archives youtube videos and then uses torrents to distribute the media to others, with the more people who took a youtube url and generated the same `youtube-dl` file hash from it, then the more verified/trusted the result is. Such can also apply to tweet json data. Or anything else. Such that we can backup, distribute, and verify our content that we contribute to these big tech companies, that they can delete or censor at any time - gaslighting us for what we actually remember - did that tweet actually happen? which screenshot of that tweet is actually real? what was that video that I liked, it just says it is restricted or private now?

If you'd prefer to reach out over email: b@bevry.me

**Comment by ichorid** [comment]
Thank you for your interest in our project! I wonder, how did you hear about it?

Your LinkedIn account shows you a proficient with JS and Web technologies. Currently, we're developing experimental support for [web-based content](https://github.com/Tribler/tribler/issues/5914) in Tribler Channels. Your skills would be *very* useful if you consider joining the effort.

However, we cannot (easily) pay outsourcers. You should talk directly to @synctext about ways of compensation for your labour.

**Comment by synctext** [comment]
@balupton Thank you for contacting us and your fascinating offer!
Please have a look at this funding option: https://nlnet.nl/propose/
We should be able to provide you with an endorsement letter from our University. We managed to obtain funding ourselves there in the past.

One idea would be to focus on "Next-generation Internet: user-driven critical infrastructure improvement".
You operate a channel with Creative Commons content for a speciality community and collect structured feedback. Usability improvements and user experience improvements in the forms of web-based interactive mock-ups?

**Comment by balupton** [comment]
This is wonderful, thank you both.

---

> Your LinkedIn account shows you a proficient with JS and Web technologies. Currently, we're developing experimental support for web-based content in Tribler Channels. Your skills would be very useful if you consider joining the effort.

Great, yep!

> One idea would be to focus on "Next-generation Internet: user-driven critical infrastructure improvement".
You operate a channel with Creative Commons content for a speciality community and collect structured feedback. Usability improvements and user experience improvements in the forms of web-based interactive mock-ups?

Not sure if it was intended or not, but that is where my prior work is. 😀 I've been iterating on a platform and community for studying research, scheduling discussions, and collecting feedback for the past several years. Plenty of iteration to continue, as is why I'm following Tribler so closely.

- original iteration was a [study group](https://coda.io/d/Bevrys-Projects_dgj0C_T-2Py/Jordan-B-Peterson-Community_su4d_#_luFd1) and [reading group](https://coda.io/d/Bevrys-Projects_dgj0C_T-2Py/Reading-Group_suBzS#_lubfw) organised via social media and google docs
- [current iteration is a discourse forum](https://discuss.bevry.me/t/about-the-study-category/991) for collaborating on semi-structured lecture notes and scheduling meetings around them, with [youtube livestreams](https://bevry.me/youtube)
- [current development iteration](https://coda.io/d/Bevrys-Projects_dgj0C_T-2Py/Bevry-Study_supir#_luVKY) is a platform that provides a youtube-like interface for watching youtube lectures, with clickable transcriptions, and incorporation of our previous study notes
- [eventual iteration](https://coda.io/d/Bevrys-Projects_dgj0C_T-2Py/Fountain_su58a#_lu5wr) is a platform for p2p video and note sharing, which is where I'm hoping Tribler would come in

---

I'll read up on the provided links.

Could either of you schedule a time for call https://balupton.com/meet to walk me onboarding of this grant (asking for grant money is new to me) and I also demo what I've been working on, and we can talk about what I could assist with.

We can also follow up via email if you prefer that b@bevry.me

**Reply by ichorid** [reply]
I've created an [issue for you](https://github.com/Tribler/tribler/issues/5935) and invited you to our GitHub organization and into our Slack chat.
Feel free to assign yourself to the issue, if you dare to touch it :wink:
